source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOHostConfig.mm

```diff
@@ -31,6 +31,17 @@ - (instancetype)initWithConfig:(std::shared_ptr<HostConfig> const &)config
     self = [super init];
     if(self && config){
         _config = config;
+        // check if requested font family name is supported by iOS, if so save it for future uses
+        NSString *requestedFontFamilyName = [NSString stringWithCString:_config->fontFamily.c_str() encoding:NSUTF8StringEncoding];
+        for(NSString* fontFamilyName in UIFont.familyNames){
+            if([fontFamilyName isEqualToString:requestedFontFamilyName]){
+                _fontFamilyName = fontFamilyName;
+            }
+        }
```

Comment:
You can say `if ([UIFont.familyNames containsObject:requestedFontFamilyName])`

Reply:
Thanks

---
In reply to: [172293124](https://github.com/Microsoft/AdaptiveCards/pull/1207#discussion_r172293124) [](ancestors = 172293124)